scripts/custom_lint_checks.py

```diff
@@ -18,23 +18,14 @@
 from __future__ import division
 from __future__ import print_function
 
-import os
-import sys
-
+import _check_docs_utils as utils
```

Comment:
I don't think we need to follow their convention here. We can just call our file docstrings_checker.py or similar and do `import docstrings_checker`.

In general try to avoid aliasing (import X as Y) where possible.